.gitlab-ci.yml

```diff
@@ -2,6 +2,8 @@ variables:
   CRAN_MIRROR: "https://cloud.r-project.org"
   _R_CHECK_FORCE_SUGGESTS_: "false"
   _R_CHECK_NO_STOP_ON_TEST_ERROR_: "true"
+  # Block truncation of any error messages in R CMD check
+  _R_CHECK_TESTS_NLINES_: 0
```

Comment:
@MichaelChirico maybe we should add it to the biggest build only? so the cran-like builds will remain consistent about log output, and eventually will help to detect that we are not displaying all informations that we might want to see

Reply:
that makes sense. CRAN already has this set to 0 though right?

anyway I'm not as knowledgeable on which would count as "biggest" so I leave that to you to add

Reply:
according to documentation CRAN does not alter that env variable

Reply:
hmm, I was going off Matt's comment here

https://github.com/Rdatatable/data.table/pull/4106/files#diff-02e6dee04136eb470c5f4a0f9f40dacbR129

Reply:
It is possible that output is logged fully till it reaches the exception, and there the 13 lines limit is applied